### ISSUE TITLE
send after adding new volumes results in "Local and archive metadata differ" on wyng 0.3.0wip 20210829

### ISSUE THREAD

**Issue by tlaurion** [tasket/wyng-backup#93]
To replicate. Works:

1. time sudo /usr/sbin/wyng --meta-dir=/var/lib/2TB-wyng-backups --from=internal:/ --subdir=media/backups/QubesOS/ arch-init
2. sudo time nice /usr/sbin/wyng --meta-dir=/var/lib/2TB-wyng-backups send --dedup

Doesn't work from here if:

1. sudo /usr/sbin/wyng --meta-dir=/var/lib/2TB-wyng-backups add vm-debian-10-chrome-zoom-root
2. sudo time nice /usr/sbin/wyng --meta-dir=/var/lib/2TB-wyng-backups send --dedup

Results in:
```
Acquiring deltas.
Error: Local and archive metadata differ.
```

3. sudo time nice /usr/sbin/wyng --meta-dir=/var/lib/2TB-wyng-backups monitor --remap --clean
4. sudo time nice /usr/sbin/wyng --meta-dir=/var/lib/2TB-wyng-backups send --dedup

Still `Error: Local and archive metadata differ`

So basically, I can continue to send changes on previously backed up volumes, but cannot send new volumes. I have to arch-init from the archive to send only changes on those volumes.

Unfortunately, i'm stuck into using 0.3x because of large and old archive. Would have to restore volumes and back them up with newer version, which I have no idea when I will be able to do for actual workstation.

**Comment by tlaurion** [comment]
Had to delete broken archive and send fresh and it worked. Not sure what happened there, but everything is fine now.
